### PR TITLE
[FAB-18092] Add example for individual orderer value updates

### PR DESCRIPTION
**Pull request by stephyee**
#### Type of change

- Documentation update

#### Description

Showcase a few individual orderer value updates

#### Related issues
[FAB-18092](https://jira.hyperledger.org/browse/FAB-18092)

### CODE REVIEW

File: configtx/example_test.go [hunk 1]
```diff
@@ -352,6 +352,33 @@ func Example_organizations() {
...
+	baseConfig := fetchChannelConfig()
+	c := configtx.New(baseConfig)
+	o := c.Orderer()
+
+	err := o.BatchSize().SetMaxMessageCount(500)
+	if err != nil {
+		panic(err)
+	}
+
+	err = o.AddConsenter(orderer.Consenter{Address: orderer.EtcdAddress{
```
Comment (wlahti) [inline configtx/example_test.go:366]:
```suggestion
	err = o.AddConsenter(orderer.Consenter{
	    Address: orderer.EtcdAddress{
```
Formatting nit: new line for Address

Comment (wlahti) [inline configtx/example_test.go:366]: Formatting nit: new line for Address and after Port: 7050

File: configtx/example_test.go [hunk 2]
```diff
@@ -352,6 +352,33 @@ func Example_organizations() {
...
+		Host: "host1",
+		Port: 7050},
+		ClientTLSCert: generateCert(),
+		ServerTLSCert: generateCert(),
+	})
+	if err != nil {
+		panic(err)
+	}
+
+	err = o.EtcdRaftOptions().SetElectionInterval(50)
```
Comment (wlahti) [inline configtx/example_test.go:376]: Hmmm, what exactly does an election interval of 50 mean? Seems odd that fabric-config and/or fabric doesn't use a duration.

File: configtx/example_test.go (hunk 1 above)
Comment (stephyee) [reply configtx/example_test.go]: 🧹

File: configtx/example_test.go (hunk 2 above)
Comment (stephyee) [reply configtx/example_test.go:374]: Election tick is not a duration it's the number of Node.Tick invocations that must pass between elections per [docs](https://hyperledger-fabric.readthedocs.io/en/latest/raft_configuration.html?highlight=electiontick#channel-configuration).

File: configtx/example_test.go [hunk 3]
```diff
@@ -352,6 +352,31 @@ func Example_organizations() {
 	o.RemoveOrganization("OrdererOrg2")
 }
 
+// This example shows updating the individual orderer configuration values.
+func ExampleOrdererGroup_SetConfiguration_individual() {
```
Comment (caod123) [inline configtx/example_test.go:356]: not too keen on the name for this example, but it's probably fine

**Pull request merged**